src/tests/unit/tests/common/components/cards/__snapshots__/cards-visualization-modifier-buttons.test.tsx.snap

```diff
@@ -1,30 +1,104 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`ExpandCollapseOnlyModifierButtons renders per snapshot 1`] = `
-<div
-  className="cardsVisualizationModifiersContainer"
->
-  <ExpandCollapseAllButton
-    allCardsCollapsed={true}
-    cardSelectionMessageCreator={{}}
-    visualHelperEnabled={true}
-  />
-</div>
+<DocumentFragment>
+  <div
+    class="cardsVisualizationModifiersContainer"
+  >
+    <button
```

Comment:
I would recommend mocking the `ExpandCollapseAllButton` component for this test so there is less irrelevant HTML in the snapshot.

Reply:
Done, please review on the latest commit.